connector/src/main/java/org/geysermc/connector/network/translators/inventory/AnvilInventoryTranslator.java

```diff
@@ -114,4 +117,30 @@ public void translateActions(GeyserSession session, Inventory inventory, List<In
 
         super.translateActions(session, inventory, actions);
     }
+
+    @Override
+    public void updateSlot(GeyserSession session, Inventory inventory, int slot) {
+        if(slot >= 0 && slot <= 2){
```

Comment:
Needs a space after the `if` and before the bracket so the formatting is consistent through the project. See code below too.